Makefile

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# ARCH is still supported, but prefer BUILDARCH and TARGETARCH.
 ARCH ?= amd64
```

Comment:
We removed ARCH in docker files and makefile targets, so it's not supported after this change?
It's a good idea to consider keeping ARCH for a few releases considering backward compatibility.

Reply:
That is the goal here. See line 18 - we assign ARCH as the default TARGETARCH. This will keep supporting build commands like `make docker-build/proxy-agent ARCH=...`.